test/katex-spec.js

```diff
@@ -1590,31 +1590,38 @@ describe("An HTML font tree-builder", function() {
 
     it("should render \\textit{R} with the correct font", function() {
         const markup = katex.renderToString("\\textit{R}");
-        expect(markup).toContain("<span class=\"mord textit\">R</span>");
+        expect(markup).toContain("<span class=\"mord textit mathrm\">R</span>");
```

Comment:
It's kind of strange to see the `mathrm` class being applied to text that's inside `\textit`.  Is this necessary?